cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -124,18 +173,32 @@ type SelfHostedEtcd struct {
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// NodeConfiguration contains elements describing a particular node
+// NodeConfiguration contains elements describing a particular node.
```

Comment:
We should also put a //TODO comment here about replacing with dynamic kubelet configuration.